src/udev/net/link-config.c

```diff
@@ -937,6 +937,49 @@ static int link_apply_sr_iov_config(Link *link, sd_netlink **rtnl) {
         return 0;
 }
 
+static int link_apply_rps_cpu_mask(Link *link) {
+        _cleanup_free_ char *mask_str = NULL;
+        LinkConfig *config;
+        int r;
+
+        assert(link);
+        config = ASSERT_PTR(link->config);
+
+        /* Skip if the config is not specified. */
+        if (!config->set_rps_cpu_mask)
+                return 0;
```

Comment:
I do not think it is necessary to introduce the flag. How about to skip to configure RPS when the CPU set is empty?

Reply:
I also think the same first, but then, how to unset if the device was configured previously with RPS enabled, without disabling or unplugging it?